<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportListino</class>
 <widget class="QWidget" name="ReportListino">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stampa listino</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>fornitore:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fornitoreComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Listino</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="listinoPrintAll">
        <property name="text">
         <string>Stampa listino completo</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="listinoPrintCurrentDate">
        <property name="text">
         <string>Stampa listino con data arrivo odierna</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Chiudi</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="printButton">
       <property name="text">
        <string>Stampa</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ReportListino</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>printButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>ReportListino</receiver>
   <slot>print()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fornitoreComboBox</sender>
   <signal>activated(QString)</signal>
   <receiver>ReportListino</receiver>
   <slot>updateFornitore(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listinoPrintAll</sender>
   <signal>clicked()</signal>
   <receiver>ReportListino</receiver>
   <slot>setReport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listinoPrintCurrentDate</sender>
   <signal>clicked()</signal>
   <receiver>ReportListino</receiver>
   <slot>setReport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>print()</slot>
  <slot>setReport()</slot>
  <slot>updateFornitore(QString)</slot>
 </slots>
</ui>
